spec/services/sign_up_service_spec.rb

```diff
@@ -0,0 +1,120 @@
+require 'spec_helper'
+
+describe SignUpService, :vcr do
+
+  let(:user_attrs) { Fabricate.attributes_for(:user) }
+  before do
+    ActionMailer::Base.deliveries.clear
+  end
+
+  context 'user data correct and credit card correct' do
+
+    include_context 'credit card charge submitted'
+    let!(:sign_up_service) do
+      user = User.new user_attrs
+      SignUpService.new(user).sign_up(stripeToken: 'token_123')
+    end
+
+    it 'is successful' do
+      expect(sign_up_service.successful?).to be(true)
+    end
+
+    it 'sets the message' do
+      expect(sign_up_service.message).to eq 'You have been charged successfully. An email has been sent to your email. Enjoy Myflix!'
+    end
+
+    it 'creates a new user' do
+      expect(User.count).to be(1)
+      expect(User.last.email).to eq(user_attrs[:email])
+      expect(User.last.full_name).to eq(user_attrs[:full_name])
+    end
+
+    it 'sends a confirmation mail' do
```

Comment:
The next two specs can't pass if this one would fail...I would be okay not having this spec. It is a bit of an extra step but not too bad if you feel better having it here.